subprojects/common/elastic/src/main/kotlin/com/avito/android/elastic/RoundRobinIterable.kt

```diff
@@ -0,0 +1,13 @@
+package com.avito.android.elastic
+
+internal class RoundRobinIterable<out T>(private val collection: Iterable<T>) {
```

Comment:
looks strange because I can't see any round-robin details there

Reply:
why so? If iterator ended we start from the beginning

Reply:
Let rename it to the `RoundRobin` because it's not an iterable

Reply:
removed it along with ability to provide multiple endpoints